auctions/tests/model_tests.py

```diff
@@ -399,3 +409,72 @@ def test_no_votes_not_counted(self, mock_send_mail):
         mock_send_mail.is_callable().times_called(0)
         mommy.make(Vote, claim=self.claim, user=self.user2, approved=False)
         mommy.make(Vote, claim=self.claim, user=self.user3, approved=True)
+
+
+class OfferTest(TestCase):
+
+    def setUp(self):
+        self.user1 = mommy.make(settings.AUTH_USER_MODEL,
+                                email='user1@test.com')
+        self.url = 'http://test.com/bug/123'
+        self.issue = mommy.make(Issue, url=self.url)
+        self.bid = mommy.make(
+            Bid, user=self.user1,
+            url=self.url, issue=self.issue, offer=50)
+
+    def test_key_created(self):
+        offer = mommy.make(Offer, bid=self.bid)
+        self.assertIsNotNone(offer.transaction_key)
+
+    @fudge.patch('stripe.Charge.create')
+    def test_bid_fees(self, mock_stripe_create):
```

Comment:
These tests depend on the django signal receiver framework. A more ideal way to test would be:
```
def test_bid_fees(self, mock_stripe_create):
    get_payment_for_offer(Bid, self.bid)
    # the rest of the assertions
```